src/main/java/Duke.java

```diff
@@ -1,10 +1,130 @@
+        }
+        System.out.println("\t" + LINE);
+    }
+
+    public static void printTask(Task task) {
+        System.out.println("\t" + LINE);
+        System.out.println("\t Got it. I've added this task:");
+        System.out.println("\t  " + task);
+        System.out.println("\t Now you have " + counter + " tasks in your list.");
+        System.out.println("\t" + LINE);
+    }
+
+    public static void printBye() {
+        System.out.println("\t" + LINE);
+        System.out.println("\t Bye! Do let me know if you need any further assistance");
+        System.out.println("\t" + LINE);
+    }
+    public static void markTaskAndPrint(int taskIndex) {
+        if(taskIndex < 0  || taskList[taskIndex] == null) {
+            System.out.println("\t" + LINE);
+            System.out.println("\t Please ensure that you enter the correct task number");
+            System.out.println("\t" + LINE);
+            return;
+        }
+        taskList[taskIndex].isDone = true;
+        System.out.println("\t" + LINE);
+        System.out.println("\t Nice, I have marked this task as done: ");
+        System.out.println("\t [" + taskList[taskIndex].getStatusIcon() + "] " +  taskList[taskIndex].description);
+        System.out.println("\t" + LINE);
+    }
+
+    public static void unmarkTaskAndPrint(int taskIndex) {
+        if(taskIndex < 0 || taskList[taskIndex] == null) {
+            System.out.println("\t" + LINE);
+            System.out.println("\t Please ensure that you enter the correct task number");
+            System.out.println("\t" + LINE);
+            return;
+        }
+        taskList[taskIndex].isDone = false;
+        System.out.println("\t" + LINE);
+        System.out.println("\t Ouch, I have unmarked this task: ");
+        System.out.println("\t [" + taskList[taskIndex].getStatusIcon() + "] " +  taskList[taskIndex].description);
+        System.out.println("\t" + LINE);
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printGreeting();
+        String input;
+        Scanner in = new Scanner(System.in);
+
+        while(true) {
+            input = in.nextLine();
+
+            if (input.equals("bye")) {
+                printBye();
+                break;
+            }
+            if (input.equals("list")) {
+                printList();
+                continue;
+            }
+
+            int spaceIndex=  input.indexOf(' ');
+            String firstWord = input.substring(0,spaceIndex);
+
+            if (firstWord.equals("mark")){
+                int indexToMark = Integer.parseInt(input.substring(5))-1; //convert to 0-based index
```

Comment:
It would be better to handle cases when the input is not an integer (To catch the exception).